person.rb

```diff
@@ -23,3 +30,26 @@ def generate_id
     @id = rand(1...1000)
   end
 end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
```

Comment:
- **[OPTIONAL]** Kindly move this class to its own file

Reply:
@DammyShittu I did that from the start but it doesn't work even tho I used the require_relative to import the nameable file so I'm having trouple there.